{
    "data_version": "4.0",
    "data_type": "CVE",
    "data_format": "MITRE",
    "CVE_data_meta": {
        "ID": "CVE-2011-3587",
        "ASSIGNER": "secalert@redhat.com",
        "STATE": "PUBLIC"
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "Unspecified vulnerability in Zope 2.12.x and 2.13.x, as used in Plone 4.0.x through 4.0.9, 4.1, and 4.2 through 4.2a2, allows remote attackers to execute arbitrary commands via vectors related to the p_ class in OFS/misc_.py and the use of Python modules."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "n/a"
                    }
                ]
            }
        ]
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "n/a",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "n/a",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_affected": "=",
                                            "version_value": "n/a"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "references": {
        "reference_data": [
            {
                "url": "http://plone.org/products/plone-hotfix/releases/20110928",
                "refsource": "MISC",
                "name": "http://plone.org/products/plone-hotfix/releases/20110928"
            },
            {
                "url": "http://plone.org/products/plone-hotfix/releases/20110928/PloneHotfix20110928-1.0.zip",
                "refsource": "MISC",
                "name": "http://plone.org/products/plone-hotfix/releases/20110928/PloneHotfix20110928-1.0.zip"
            },
            {
                "url": "http://plone.org/products/plone/security/advisories/20110928",
                "refsource": "MISC",
                "name": "http://plone.org/products/plone/security/advisories/20110928"
            },
            {
                "url": "http://pypi.python.org/pypi/Products.PloneHotfix20110928/1.0",
                "refsource": "MISC",
                "name": "http://pypi.python.org/pypi/Products.PloneHotfix20110928/1.0"
            },
            {
                "url": "http://secunia.com/advisories/46221",
                "refsource": "MISC",
                "name": "http://secunia.com/advisories/46221"
            },
            {
                "url": "http://secunia.com/advisories/46323",
                "refsource": "MISC",
                "name": "http://secunia.com/advisories/46323"
            },
            {
                "url": "http://zope2.zope.org/news/security-vulnerability-announcement-cve-2011-3587",
                "refsource": "MISC",
                "name": "http://zope2.zope.org/news/security-vulnerability-announcement-cve-2011-3587"
            },
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=742297",
                "refsource": "MISC",
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=742297"
            }
        ]
    }
}